src/routes/conversation/[id]/+server.ts

```diff
@@ -379,6 +379,8 @@ export async function POST({ request, locals, params, getClientAddress }) {
 					webSearch: webSearch ?? false,
 					toolsPreference: toolsPreferences ?? {},
 					promptedAt,
+					userId: userId.toString(),
```

Comment:
I think what zero-gpu wants it the username cc @cbensimon 

so `locals.user?.username` - and no need for fallbacks, the fallback is the user's IP

Reply:
handled in 2adf0fb97b3ef759453c75eacebc37318110ab87